benchmark.rb

```diff
@@ -0,0 +1,14 @@
+# To set up the test, replace in facebook_data_analyzer/analyze_facebook_data.rb the following line:
```

Comment:
@marzann This would also work, without changing `analyze_facebook_data.rb`
```
require 'benchmark'
require_relative 'analyze_facebook_data'

Benchmark.bm do |x|
  x.report { Workbook.new(catalog: ARGV[0]) }
end
```